core/src/main/scala/org/apache/pekko/persistence/jdbc/config/PekkoPersistenceConfig.scala

```diff
@@ -221,7 +221,10 @@ class DurableStateTableConfiguration(config: Config) {
   val batchSize: Int = config.getInt("batchSize")
   val schemaName: Option[String] = cfg.asStringOption("schemaName")
   val columnNames: DurableStateTableColumnNames = new DurableStateTableColumnNames(config)
-  val stateSequenceConfig = DurableStateSequenceRetrievalConfig(config)
+  val stateSequenceConfig: DurableStateSequenceRetrievalConfig = DurableStateSequenceRetrievalConfig(config)
+  private val schemaPrefix: String = schemaName.fold("")(schema => s"$schema.")
+  // @since "1.1.1"
```

Comment:
```
/**
 * @since 1.1.1
 */
```

Reply:
this scaladoc since tag is supported by IDEs and appears in the genearted scaladocs